introduction.md

```diff
@@ -0,0 +1 @@
+It's me, Mario! Ooops, sorry...It's me, Lucas!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.